docs/source/examples/workspaces/openshift/PinFile

```diff
@@ -1,39 +1,42 @@
 ---
-openshift-new:
```

Comment:
@p3ck I am not sure why this Pinfile and workspace is modified . 
The intention of older Pinfile is to use oepnshift as a resource provider . 
According to that logic this workspace should go under openstack folder . 
@p3ck @herlo Any thoughts ???

Reply:
@samvarankashyap the workspaces bits are not being used in documentation yet, but that would be the intent eventually. 

@p3ck I do somewhat agree with @samvarankashyap here that this workspace should maybe be its own, rather than lumping it in with another. Thoughts?